extension/js/common/core/common.ts

```diff
@@ -288,7 +288,7 @@ export class Url {
     rawParamNameDict[urlParamName] = urlParamName;
     rawParamNameDict[Url.snakeCaseToCamelCase(urlParamName)] = urlParamName;
     rawParamNameDict[Url.camelCaseToSnakeCase(urlParamName)] = urlParamName;
-    const shortened = urlParamName.replace('account', 'acct').replace('message', 'msg').replace('attachment', 'att');
+    const shortened = urlParamName.replace('account', 'acct').replace('message', 'msg');
```

Comment:
I haven't looked at the tests yet, but if they broke, this could be the cause - it would be ok to leave this one line as is.